<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperMarketWidget</class>
 <widget class="QWidget" name="SuperMarketWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="DiamondMarketBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>460</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>钻石商城</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BeanMarketBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>460</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>豆豆商城</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值1元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>10钻石</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD60">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值6元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD60">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>60钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelD300">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>300钻石</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD300">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值30元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD1280">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值128元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD1280">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>1280钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD3280">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值328元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD3280">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>3280钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD6480">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>充值648元</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD6480">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>6480钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD1280_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1280钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD10_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1000豆</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD300_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>30000豆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelD60_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>6000豆</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD6480_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>64800豆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD60_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>60钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD1280_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>12800豆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD10_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>10钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD3280_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3280钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelD3280_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string>32800豆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD6480_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6480钻石</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label6_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>320</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label4_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border: 0.5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD300_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>300钻石</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
